src/main/scripts/src/app/app.component.scss

```diff
@@ -11,7 +11,7 @@ header {
 }
 
 main {
-  width: 100vw;
-  padding: 0 5vw;
-  overflow-y: auto;
+  margin: 10px;
+  height: calc(100vh - 90px);
```

Comment:
We will probably need to change the whole base layout using Bootstrap in order to solve this. So keep it like this for now and we will fix it in the future.